### DISCUSSION TITLE
Email contact form

### DISCUSSION THREAD

**Discussion by zak2002** [requarks/wiki#5145] [Help / Questions] [answered]
Hi all,
I added an html/css contact form with fields say Name, Email and Message. But I am unable to figure out how to send it. The Email is configured already and works. I just need to figure out what to mention in the html form code tag "action=". Does the website allow to add a php code, so that I can generate a page say sendmail.php, put it in the root folder and put the action tag as "action=\wiki\sendmail.php"? Any alternatives if this is not possible?

ps: I am not asking about mailto links where visitor's email client is called.

Thanks in advance.

zak

**Comment by zak2002** [comment]
No hints at all?

**Comment by NGPixel** [comment, marked as answer]
What you're asking is not possible. You can have a form on Wiki.js but you'd need to point to an external server to handle the form and send the mail, not Wiki.js
